### PR TITLE
data sherpa job ad

### PR DESCRIPTION
**Pull request by nefiger**
Could someone take a quick peek for any obvious spelling issues etc and if all is well, merge please? :)

thanks!

@xybrnet @longhotsummer @adieyal

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -19,12 +19,36 @@
...
 	  </ul>
 	</nav>
       </div>
 
-      <div class="col-sm-9">
+      <div class="col-sm-9">	
+	<section>
+	  <h2><a id="datasherpa">Data Journalist / Sherpa</a></h2>
+
+	  <p>Help launch Africa’s first data journalism cadet school. Starting in February 2016, the cadet school will run three months of intensive coursework as well as an active data-driven newsroom. We are looking for an experienced news editor  to run the newsroom fill this key leadership post.</p>
```
Comment (longhotsummer) [inline careers.html]: I always think job ads should start with "We're looking for X". Then dig into the details.

File: careers.html [hunk 2]
```diff
@@ -19,12 +19,36 @@
...
+      <p>In addition to training, the cadet school will be a centre for news innovation. Through the newsroom, we will experiment with different approaches to delivering content to audiences with an emphasis on actionable news. We are actively involved in ongoing projects that promote innovation in the media space through the use of data-driven storytelling and the cadet school will play an important role in achieving this.</p>
+      <p>As part of our newsroom team driving this process, the successful applicant will be involved in the following:
```
Comment (longhotsummer) [inline careers.html]: You'll play a key part of our newsroom team and be expected to:

File: careers.html [hunk 3]
```diff
@@ -19,12 +19,36 @@
...
-      <div class="col-sm-9">
+      <div class="col-sm-9">	
+	<section>
+	  <h2><a id="datasherpa">Data Journalist / Sherpa</a></h2>
+
+	  <p>Help launch Africa’s first data journalism cadet school. Starting in February 2016, the cadet school will run three months of intensive coursework as well as an active data-driven newsroom. We are looking for an experienced news editor  to run the newsroom fill this key leadership post.</p>
+      <p>In addition to training, the cadet school will be a centre for news innovation. Through the newsroom, we will experiment with different approaches to delivering content to audiences with an emphasis on actionable news. We are actively involved in ongoing projects that promote innovation in the media space through the use of data-driven storytelling and the cadet school will play an important role in achieving this.</p>
+      <p>As part of our newsroom team driving this process, the successful applicant will be involved in the following: 
+	  <ul>
+	    <li>Share your knowledge and previous experience from other newsrooms in helping Code for South Africa establish a data-driven newsroom</li>
```
Comment (longhotsummer) [inline careers.html]: "in helping" -> "to help"

File: careers.html [hunk 4]
```diff
@@ -19,12 +19,36 @@
...
+	    <li>Assume the role of a data sherpa overseeing the day-to-day editorial management (effectively news editing) role. You will report to the editor, who will have overall oversight of the newsroom's activities</li>
```
Comment (longhotsummer) [inline careers.html:37]: "Oversee the day-to-day news editorial management".

Is this person a data wrangler, a news editor, or both?

File: careers.html [hunk 5]
```diff
@@ -19,12 +19,36 @@
...
+	    <li>Compile and oversee a news diary with short, medium and long term data-driven stories/news projects, working with data wranglers, coders, analysts and cadets</li>
+	    <li>Commit to a schedule to publish work in the local, national and international press</li>
+	    <li>Participate in the teaching of short modules in our data journalism cadet school</li>
+	    <li>Provide technical support to the team where appropriate (data wrangling, visualisations, etc)</li>
+	    <li>Work with cadets to explore datasets for potential stories and help decide on the way these stories will be covered</li>
+	    <li>Help pioneer new approaches to telling stories, especially in a South African context. This will include exploring news ways of personalising content, as well as delivery of content to low-powered mobile devices</li>
```
Comment (longhotsummer) [inline careers.html]: "new**s** ways"

File: careers.html [hunk 6]
```diff
@@ -19,12 +19,36 @@
...
+	    <li>Participate in exploring new funding models for stories</li>
+	    <li>You may be involved in other Code for South Africa projects which relate to open data</li>
```
Comment (longhotsummer) [inline careers.html:45]: "Be involved with"... no need to be ambiguous.

File: careers.html (hunk 4 above)
Comment (nefiger) [inline careers.html:37]: A bit of both. Ray and Adi collaborated on the description and wording to get to something that should appeal to the correct profile.

File: careers.html (hunk 5 above)
Comment (nefiger) [inline careers.html]: topical - no? ;)

File: careers.html (hunk 6 above)
Comment (nefiger) [inline careers.html:45]: Are you saying remove the 'may'? That implies they will definitely be involved with other projects. Can we say this for certain at this stage?

**Comment by nefiger** [comment]
merged @xybrnet's changes and implemented @longhotsummer's ones

**Comment by longhotsummer** [comment]
@nefiger a bunch of comments I made applied to the previous diff, so maybe you missed them?

**Comment by longhotsummer** [comment]
I suggest trying to avoid starting all sentences with "Subclause, main clause." You can generally re-word that to be simpler and more direct. 

eg. "Starting in February 2016, the cadet school will run three months of intensive coursework" -> "The cadet school starts in February 2016 and will run three months..."

**Comment by nefiger** [comment]
uuh @longhotsummer I did see those and merged Damian's stuff in too, just forgot to push. I've done so now :blush:

**Comment by longhotsummer** [comment]
:+1:

**Pull request merged**